newswires/client/src/App.tsx

```diff
@@ -107,15 +103,18 @@ export function App() {
 		} else {
 			document.title = 'Newswires';
 		}
-	}, [isPoppedOut, config.query]);
+	}, [isTickerView, config.query]);
 
-	const largeMinBreakpoint = useEuiMinBreakpoint('l');
-	const largeMaxBreakpoint = useEuiMaxBreakpoint('l');
+	const errorPromptComponent = useMemo(() => {
```

Comment:
Just checking my understanding here. This component here will store an error based on whether there has been an error. If the status changes, this component would remain populated? 

It looks like there is an explicit check for the `status` where it is used further down, so I guess it would be fine for this component to remain populated even after a state change. Just checking the behaviour